article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
+
+After adding explanatory and effect modifying variables to the graph, we turn our attention to mediating variables.
+Again, we iterate through each explanatory variable.
+On each iteration, we brainstorm all the variables through which our current explanatory variable influences the outcome.
+For instance, consider how the presence of a bike lane influences bicycle mode choice.
+We hypothesize that an individual's subjective perception of safety is be the primary (or sole) variable through which bicycle lane presence influences mode choice.
+Accordingly, we add subjective perception of safety to our causal graph for travel mode choice.
+
+Coming to our second to last category of variables, we think of confounding variables.
+The process is similar to how we generated effect modifying variables.
+We iterate through each of the explanatory, mediating, and effect modifying variables, thinking specifically of any variables that both cause the current variable in the iteration and cause the outcome variable(s).
+We call these variables, which cause our outcome and current variables in the iteration, confounding variables.
+We will add each of them to our causal graph.
+
+For the last set of variables, we should explicitly consider the role of time, even in research that may be cross-sectional due to the data that is available to us.
+In reality, how do we think our system evolves over time?
+If we consider multiple observations of a given decision maker, how do variables observed of that decision maker at time $t$ partially cause future variables important to the context or outcomes for that decision maker at time $t+1$?
+How do the actions of a decision maker $i$ at time $t$ partially cause the future context or outcomes of a decision maker $j$?
+We should add explicit nodes to our graph, subscripted or denoted by time, to show the cross-time causal relationships in our system.
+
+At this point, we have added all the outcome, explanatory, effect modifying, mediating, confounding, and time-indexed variables that we can think of to our causal graph.
+They are all disconnected nodes, singletons, in the graph.
+We now focus on pruning nodes from this graph, before drawing our final hypothesized connections.
+In particular, we focus on pruning ``post-outcome'' variables that are not part of the causal graph for future time periods or other observations.
+The reason for this is that conditioning on such post-outcome variables biases one's causal effect estimates.
+
+To remove the problematic variables, we iterate through each of the non-outcome variables in our graph, and we assess whether the variable is actually a result of the outcome (perhaps in combination with other variables in our graph).
+These post-outcome variables temporally follow the outcome variable(s) but do not cause variables in the causal graph for other observations.
+We remove all such post-outcome variables from our graph.
```

Comment:
```suggestion
To remove the problematic variables, we iterate through each of the non-outcome variables in our graph, and we assess whether a variable is actually a result of the outcome (perhaps in combination with other variables in our graph).
These post-outcome variables temporally follow the outcome variable(s) but do not cause or affect variables in the causal graph for other observations.
We remove all such post-outcome variables from our graph.
```

Reply:
What sort of affects do you have in mind? I think of affect and cause as synonyms in this case, and I'm not immediately picturing a way that variable A affects variable B without causing variable B.

Reply:
My suggestion was because when I use "cause" I use it as "cause a change in a variable" as opposed to cause a variable. It's just how we're reading things, not so much the interpretation of how post-outcome variables affect variables in causal graphs for other observations.